docs/tutorials/services/data-mlmodel-tutorial.md

```diff
@@ -0,0 +1,322 @@
+
+   {{< alert title="Tip" color="tip" >}}
+
+   For best results:
+
+   - Provide at least 10 images of the same object, taken from different angles, and repeat this approach for each object you want your smart machine to be able to identify.
+   - Include some images that do not contain any of the objects you wish to identify, but do not tag these images.
+   - If your smart machine operates in various lighting conditions, such as changing sunlight, include images of each object from varying lightning conditions.
+   - You can tag a single image with multiple tags if needed, but be sure to use `Multi label classification` when training your model later in this tutorial.
+
+   {{< /alert >}}
+
+Feel free to return to your smart machine's **Control** tab to position your camera to capture additional images from a variety of different angles, or with different lighting or background compositions.
+Generally, the more different perspectives of a given object you tag, the more likely your model will be able to identify it, even under differing conditions.
+The following is an example of a good selection of images containing the `blue_star` tag, taken from a variety of angles:
+
+{{< imgproc src="/tutorials/data-management/filter-by-blue-star.png" alt="The data tab showing images that contain blue stars from various angles, with a search filter for blue_star applied to only show images tagged with that label" resize="1200x" >}}
+
+If you want to remove a tag, click the **X** icon to the right of the tag name below the **Tags** field.
+
+For more information, see [Train a model](/manage/ml/train-model/).
+
+### Filter images using tags
+
+Once you have tagged your images, you can use those tags to filter your images to control which images to run your model on.
+
+To filter images by tag, use the **Tags** drop down filter on the left-hand side to select the tag you want to filter by, then click the **Search** button to limit displayed images to only those matching the provided tag.
+
+For example, the following shows an images data set filtered to only show images tagged with the `red_star` tag:
+
+{{< imgproc src="/tutorials/data-management/filter-by-red-star.png" alt="The filtering pane on the left of the data page with the tag red_star checked and the search button shown" resize="250x" >}}
+
+You can search for multiple tags at once, which will display images that match any of the included tags.
+To reset a filter, and return to viewing all captured images, click the **Reset filters** button on the left, then click **Search** again.
+
+### Train a new model based on a filter
+
+Filtering images using tags also constrains the model you train to only those images you want to inform your model.
+This is especially useful when you may have captured a large number of images of a variety of objects, but only want to train a model to identify a few of them.
```

Comment:
hmm I don't like that there's no images that don't cotnain the objects . I think it's better to suggest filtering by robot and time and then you select the labels for training on the right